_posts/2023-09-26-Hacktoberfest-2023.adoc

```diff
@@ -0,0 +1,53 @@
+
+. *WildFly Elytron:* WildFly Elytron is a new WildFly sub-project that is completely replacing the combination of PicketBox and JAAS as the WildFly client and server security mechanism.
+* Here is the https://github.com/wildfly-security/wildfly-elytron[GitHub repository], https://github.com/wildfly-security/wildfly-elytron/blob/2.x/CONTRIBUTING.md[Contributing Guide], and https://issues.redhat.com/browse/ELY-2636?filter=12383825[Hacktoberfest Issues link].
+
+. *Ansible Middleware:* Ansible Middleware is a suite of supported content that combines the power of Ansible automation with Red Hat's Middleware portfolio. It allows organizations to meet their business demands and automate the installation, configuration, and management of Enterprise Middleware.
+* Here is the https://github.com/ansible-middleware[GitHub repositories], https://github.com/ansible-middleware/keycloak/blob/main/CONTRIBUTING.md[Contributing Guide], and https://github.com/search?q=org%3Aansible-middleware+state%3Aopen+label%3Ahacktoberfest&type=issues[Hacktoberfest Issues link].
+
+. *Undertow:* Undertow is a versatile and high-performance web server that provides both blocking and non-blocking APIs based on NIO. Its modular architecture allows you to build a web server tailored to your needs, from a full Java EE servlet 4.0 container to a low-level non-blocking handler.
+* Here is the https://github.com/undertow-io/undertow[GitHub repository], https://github.com/undertow-io/undertow/blob/master/CONTRIBUTING.md[Contributing Guide], and Hacktoberfest Issues link.
+
+. *Narayana:* Narayana is a transactions toolkit that provides support for applications using various standards-based transaction protocols. It's an essential component of the WildFly application server and continues to be developed as a standalone transaction manager.
+* Here is the https://github.com/jbosstm/narayana[GitHub repository], https://github.com/jbosstm/narayana/blob/main/CONTRIBUTING.md[Contributing Guide], and https://issues.redhat.com/browse/JBTM-2493?filter=12421681[Hacktoberfest Issues link].
+
+. *Infinispan Console:* Infinispan Console is a front-end application for the web console of the new Infinispan Server. Built using Patternfly 5 and React 18, it's a dynamic and engaging project to contribute to.
+* Here is the https://github.com/infinispan/infinispan-console[GitHub repository], Contributing Guide, and https://github.com/infinispan/infinispan-console/issues[Hacktoberfest Issues link].
+
+== How to reach out to us for Hacktoberfest related queries?
+
+Please join our https://wildfly.zulipchat.com/#narrow/stream/Hacktoberfest[Hacktoberfest Zulip chatroom] and reach out to us if you have any queries or need assistance. We’re here to ensure a smooth journey for you.
```

Comment:
Can we point separate chat room for Infinispan console https://infinispan.zulipchat.com/#narrow/stream/118645-infinispan/topic/hacktoberfest

Reply:
Currently, we are asking contributors to join a single chatroom as it'll be easy plus Zulip has a feature to create new threads so if needed contributors can create project-specific new threads and discuss their queries.

Reply:
We all can join the chatroom to drive the initiative together, providing separate chatrooms for every project will make it confusing.

Reply:
Unless @dpanshug believes people from Infinispan are committed to monitoring WildFly zulip, we shouldn't tell Infinispan contributors to go there. It's a completely different community. The WildFly developers are unlikely to have any idea about Infinispan Console. WF does not use it. There are only a couple of us who are familiar with contributing to any part of Infinispan, so even simple process-oriented questions may go unaddressed if we're relying on WildFly devs for coverage.

Reply:
After having a discussion with @dpanshug he decided not to go with the article as WildFly and its components are not much aligned with the Infinispan console and both the communities are completely different. 
So I'll remove the Infinispan console part.